<?xml version="1.0"?>
<launch>
 <node pkg="pylon_camera" name="pylon_camera_node" type="pylon_camera_node" output="screen" > <!--  respawn="true" respawn_delay="30" -->
  <param name="camera_frame" value="test_cam" type="string" /> <!-- frame to which the camera is attached -->
  <param name="exposure" value="-1" type="int" />
  <param name="use_sequencer" value="false" type="bool" />
  <param name="desired_framerate" value="30.0" type="double" />
  <param name="use_brightness" value="true" type="bool" />
  <param name="intrinsic_yaml_string" command="cat $(find pylon_camera)/calib/cam15_f6mm_2014-12-18.yml"/>
 </node>
 <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
  <remap from="image" to="/pylon_camera_node/image_raw"/>
 </node>
</launch>
